.changeset/cool-ladybugs-argue.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/core-components': minor
+'@backstage/create-app': minor
+'@techdocs/cli': minor
+'@backstage/plugin-dynatrace': minor
```

Comment:
These packages does not need a changeset as there are no changes to those packages and cypress is just an internal test concern.